index.html

```diff
@@ -61,6 +61,282 @@ <h1>i'm Adan <br>
+                </div>
+                <div class="see-projects">
+                    <button class="see">See Projects</button>
+                </div>
+
+            </div>
+
+
+
+            <div class="tonic2">
+                <div class="grid1">
+                    <img src="./images/Snapshoot Portfolio3.png" alt="snapshot">
+
+                </div>
+                <div class="details">
+                    <h1>Tonic</h1>
+
+                    <ul class="specific">
+                        <li class="images">CANOPY</li>
+                        <li id="fade">Back End Dev</li>
+                        <li id="fade">2015</li>
+                    </ul>
+
+                    <p class="brief">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+
+                    <div class="languages">
+                        <ul class="html-css">
+                            <li>
+                                <a href=""><img src="./images/html.png" class="img2" alt=""></a>
+                            </li>
+                            <li>
+                                <a href="#index.html"><img src="./images/Tag.png" class="img2" alt=""></a>
+
+                            </li>
+                            <li>
+                                <a href="#index.html"><img src="./images/Tag (1).png" class="img2" alt=""></a>
+                            </li>
+                        </ul>
+                    </div>
+                    <div class="see-projects">
+                        <button class="see">See Projects</button>
+                    </div>
+
+                </div>
+            </div>
+            <div class="grid3">
+                <img src="./images/Snapshoot Portfolio (2).png" alt="snapshot">
+
+            </div>
+
+            <div class="details">
+                <h1>Multi-Post
+                    <br>
+                    Stories</h1>
+
+                <ul class="specific">
+                    <li class="images">CANOPY</li>
+                    <li id="fade">Back End Dev</li>
+                    <li id="fade">2015</li>
+                </ul>
+
+                <p class="brief">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <div class="languages">
+                    <ul class="html-css">
+                        <li>
+                            <a href=""><img src="./images/html.png" class="img2" alt=""></a>
+                        </li>
+                        <li>
+                            <a href="#index.html"><img src="./images/Tag.png" class="img2" alt=""></a>
+
+                        </li>
+                        <li>
+                            <a href="#index.html"><img src="./images/Tag (1).png" class="img2" alt=""></a>
+                        </li>
+                    </ul>
+                </div>
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure all the projects' cards design stick as much as possible to the Figma design.

| Figma Design | Your Design |
| -------------------- | -------------------- |
| ![Screenshot from 2022-09-24 16-16-13](https://user-images.githubusercontent.com/83042742/192103139-e12f7da5-a7d6-44bc-a10e-64bc46a42826.png) | ![Screenshot from 2022-09-24 16-15-16](https://user-images.githubusercontent.com/83042742/192103153-5288b6d9-86c3-4ee6-8ad9-828cdb315b4d.png) |

Reply:
@tresorsawasawa should I add a card? am confused, I had a card yesterday and I was told to remove it. Please explain to me further.

Reply:
@tresorsawasawa are you there, please?

Reply:
Hi @AbdimulhinYussuf3675

Yes I am here

Reply:
@tresorsawasawa should i add a card ?

Reply:
@AbdimulhinYussuf3675

No, only match your design to your chosen Figma template